tests/csv_import_test.py

```diff
@@ -7,10 +12,10 @@ def test_import():
 
     for d in pkunit.case_dirs(group_prefix="conformance"):
         for f in pkio.sorted_glob("*.csv"):
-            _ = sirepo.csv.read_as_number_list(f)
+            _ = csv.read_as_number_list(f)
```

Comment:
Why bother with the underscore? And the pass below isn't needed.

Reply:
82da885